bin/build.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-
-#TODO - add rest of steps needed to build the package
-
-docker run --rm -v "$PWD:/conjurinc/api-java" -w /conjurinc/api-java maven:3-jdk-8 /bin/bash -c \
-"mvn -X -e clean package -Dmaven.test.skip=true"
\ No newline at end of file
+docker run --rm \
+  -v "$PWD:/conjurinc/api-java" \
+  -w /conjurinc/api-java maven:3-jdk-8 \
```

Comment:
I'm guessing that these two paths also needs to change

Reply:
Agreed - Fixed all three scripts to use a new path, lemme know what you think